src/ecdsa/keys.py

```diff
@@ -1133,3 +1133,22 @@ def sign_number(self, number, entropy=None, k=None):
         assert 1 <= _k < order
         sig = self.privkey.sign(number, _k)
         return sig.r, sig.s
+
+    def ecdh_get_shared_secret(self, remote_public_key):
+        """
+        Elliptic-curve Diffie–Hellman (ECDH). A key agreement protocol.
```

Comment:
this em-dash (`–`) is outside 7-bit ASCII, please add encoding definition to the file

Reply:
dash removed)